src/go/vtunnel/.gitignore

```diff
@@ -0,0 +1 @@
+*.exe
```

Comment:
This should be at the top level `.gitignore` — we never want to check in exe files. (They should either be built from source, or downloaded in one of the postinstall scripts.)